src/utilities/collection-utils.test.ts

```diff
@@ -322,6 +322,40 @@ describe("CollectionUtils", () => {
         });
     }); // end isNotEmpty
 
+    describe("#length", () => {
```

Comment:
I'm not sure how many other describe blocks are like this, but would you mind wrapping each one in a region block with the method name being tested?

```TS
    // -----------------------------------------------------------------------------------------
    // #region length
    // -----------------------------------------------------------------------------------------
    
    // #endregion length
```